src/customElements.ts

```diff
@@ -326,7 +318,7 @@ export function initializeElement(element: CustomElement) {
 			if (childrenType === ChildrenType.DOJO) {
 				children.push(w(DomToWidgetWrapper(childNode), properties));
 			} else {
-				children.push(w(DomWrapper(childNode), properties));
+				children.push(dom({ node: childNode, props: properties }));
```

Comment:
This means that only properties and no attributes or events will be added to custom element children...

Reply:
Is only a single key property anyway...